src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -259,7 +260,13 @@ private ZipFileProcessingResult processZipFileContent(
 
             envelopeProcessor.assertDidNotFailToUploadBefore(envelope.zipFileName, containerName);
 
-            Optional<OcrValidationWarnings> ocrValidationWarnings = this.ocrValidator.assertOcrDataIsValid(envelope);
+            Optional<OcrValidationWarnings> ocrValidationWarnings;
+
+            if (envelope.classification != EXCEPTION) {
+                ocrValidationWarnings = this.ocrValidator.assertOcrDataIsValid(envelope);
+            } else {
+                ocrValidationWarnings = Optional.empty();
+            }
```

Comment:
Should we maybe move this logic to the validator itself? BlobProcessorTask is already responsible for too much.